NoteFind/View Controllers/FeedViewController.m

```diff
@@ -51,4 +54,14 @@ - (IBAction)newPost:(id)sender {
     [self performSegueWithIdentifier:@"newPost" sender:nil];
 }
 
+- (IBAction)didTapLogout:(id)sender {
+    SceneDelegate *sceneDelegate = (SceneDelegate *)self.view.window.windowScene.delegate;
+    [PFUser logOutInBackgroundWithBlock:^(NSError * _Nullable error) {
+        // PFUser.current() will now be nil
```

Comment:
How are we handling if logout fails? The error message should be `nonnull` in this case and we might need to inform the user

Reply:
resolved